<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplementalListWidgetItem</class>
 <widget class="QWidget" name="SupplementalListWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>46</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>46</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>46</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #404040;

margin: 0px 0px;

QWidget::QFrame {
	border: 1px solid rgba(255, 255, 255, 0.1);
	box-sizing: border-box;
	border-radius: 8px; 
}






</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>0</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/*
display: flex;
flex-direction: row;
justify-content: center;
align-items: center;
*/
padding: 2px;

background: rgba(255, 255, 255, 0.005);
border: 1px solid rgba(255, 255, 255, 0.1);
box-sizing: border-box;
border-radius: 8px;

margin: 0px 0px;

QFrame {
	background: #404040; 
	border-radius: 6px;
}

QFrame:hover {
	background: #474747;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_Add">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>8</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background: #404040;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/SVG/svg/icon/reading-order-add-to-list.svg</normaloff>:/SVG/svg/icon/reading-order-add-to-list.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Filetype">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>8</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/* icon/resource-image */





border: 0px;

QFrame {
	background: #404040;
	border-radius: 6px;
}

QFrame:hover {
	background: #474747;
}

</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/SVG/svg/icon/resource-document.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Remove">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>8</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #404040;
border: 0px;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/SVG/svg/icon/common-remove.svg</normaloff>:/SVG/svg/icon/common-remove.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>10</y>
      <width>112</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>112</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>19</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #404040;
border: 0px;
color: #FFFFFF;


font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 19px;
margin: 0px 0px;
border: 0px;
color: #FFFFFF;
background: #404040;
border-radius: 0px;

</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
